task/golang-test/0.3/README.md

```diff
@@ -0,0 +1,65 @@
+# Golang Test
+
+This task is a Golang task to test Go projects.
+
+## Install the task
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/main/task/golang-test/0.2/golang-test.yaml
+```
+
+### Parameters
+
+* **package**: base package to build in
+* **packages**: packages to test (_default:_ ./cmd/...)
+* **context**: path to the directory to use as context (default: .)
+* **version**: golang version to use for builds (_default:_ latest)
+* **flags**: flags to use for `go test` command (_default:_ -race -cover -v)
+* **GOOS**: operating system target (_default:_ linux)
+* **GOARCH**: architecture target (_default:_ amd64)
+* **GO111MODULE**: value of module support (_default:_ auto)
+* **GOCACHE**: value for go caching path (_default:_ "")
+* **GOMODCACHE**: value for go module caching path (_default:_ "")
+* **CGOENABLED**: go env. enables cgo
+* **GOPRIVATE**: go env. private modules to be fetched from original source
+* **GIT_ALLOW_PROTOCOL**: list of vcs schemes to download module dependencies
+* **git_config_add**: git configuration to add
+* **pkgs**: linux packages to install from the apt library
+* **pkg_source**: additional linux package sources to fetch apt packages from
+
```

Comment:
Missing:

`* **GOPROXY**: [Proxy settings](https://go.dev/ref/mod#goproxy-protocol) for Go dependencies (default: empty, not enabled)`

:wink:

Reply:
thanks for pointing it out 👍